"""try:

    print(x)

except NameError:
    print("Já identificamos o erro e estamos trabalhando nisso.")"""


"""try:
    x = int(input("Digite um nome: "))
    print(x)

except ValueError:
    print("Encontramos um erro.")"""


"""try:
    x = str(input("Digite um número: "))
    y = x * x

except TypeError:
    print("Encontramos um erro, já estamos trabalhando nisso. Desculpe pelo transtorno.")"""

"""try:
    x = [1, 2, 3, 4, 5]
    print(x[5])

except IndexError:
    print("Encontramos um erro de Index, já estamos trabalhando nisso.")"""

"""try:
    x = str(input("Drive"))
    j = "".reversed(x)
    print(j)
except AttributeError:
    print("Encontramos um erro de Atributo, já estamos trabalhando nisso.")"""